roxctl/analyze/netpol/cta.go

```diff
@@ -62,10 +61,9 @@ func (cmd *analyzeNetpolCommand) ouputConnList(connsStr string) error {
 	return nil
 }
 
-func writeFile(filename string, destDir string, content string) error {
-	outputPath := filepath.Join(destDir, filename)
+func writeFile(outputPath string, content string) error {
 	if err := os.MkdirAll(filepath.Dir(outputPath), 0755); err != nil {
-		return errors.Wrapf(err, "error creating directory for file %q", filename)
+		return errors.Wrapf(err, "error creating directory for file %q", filepath.Base(outputPath))
```

Comment:
why not printing `outputPath` as the file for which there is an error creating directory for?

Reply:
not for a specific reason .... correcting